tests/test_postgres_client.py

```diff
@@ -0,0 +1,189 @@
+import datetime
+from typing import List, Any, Optional
+from unittest import TestCase
+from unittest.mock import Mock, call, patch
+from psycopg2.errors import InsufficientPrivilege  # noqa
+
+from apollo.agent.agent import Agent
+from apollo.agent.constants import (
+    ATTRIBUTE_NAME_ERROR,
+    ATTRIBUTE_NAME_RESULT,
+    ATTRIBUTE_NAME_ERROR_TYPE,
+)
+from apollo.agent.logging_utils import LoggingUtils
+
+_POSTGRES_CREDENTIALS = {
+    "host": "www.test.com",
+    "user": "u",
+    "password": "p",
+    "port": "5439",
```

Comment:
it's not relevant, but I guess this should be 5432 just to be consistent with PG default values